docs/fundamentals/code-analysis/quality-rules/ca2254.md

```diff
@@ -31,6 +31,8 @@ The logging message template should not vary between calls.
 
 Update the message template to be a constant expression.
 
+See the [LoggerExtensions.LogInformation Method](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loggerextensions.loginformation) for usage examples.
```

Comment:
```suggestion
See the [LoggerExtensions.LogInformation Method](https://docs.microsoft.com/dotnet/api/microsoft.extensions.logging.loggerextensions.loginformation) for usage examples.
```

Remove the locale from the URL, but more importantly - I'm curious about the text in this sentence as this specific API ref doesn't have any example usages.